array-manipulation-1/src/initial.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-unused-vars */
+function initial(array){
+    let newArr = [];
+
+    for(let i = 0;i<array.length;i++){

```

Comment:
You can simplify this and ditch the `if` by changing your loop's conditional expression

```js
for (let i = 0; i < array.length - 1; i++) {
```